lib/asyn-ares.c

```diff
@@ -89,8 +89,20 @@ struct ResolverResults {
   int num_pending; /* number of ares_gethostbyname() requests */
   Curl_addrinfo *temp_ai; /* intermediary result while fetching c-ares parts */
   int last_status;
+  struct curltime happy_eyeballs_dns_time; /* when this timer started, or 0 */
 };
 
+/* How long we are willing to wait for additional parallel responses after
+   obtaining a "definitive" one.
+
+   This is intended to equal the c-ares default timeout.  cURL always uses that
```

Comment:
This should be "curl", but that nitpick can be fixed by the committer rather than require a fresh rebase IMHO.

Reply:
Oops.  I've been spelling it what I guess is the old-fashioned way!  

Thanks!

BTW, maybe someone should fix the Wikipedia article's spelling :)